great_expectations/experimental/metric_repository/cloud_data_store.py

```diff
@@ -11,11 +13,41 @@
 
 
 class CloudDataStore(DataStore[StorableTypes]):
+    def __init__(self, context: AbstractDataContext):
+        super().__init__(context=context)
+        config = self._context._cloud_config
+        self._session = create_session(access_token=config.access_token)
+        self._url = (
+            f"{config.base_url}/organizations/{config.organization_id}/metric-runs"
+        )
+
+    def _construct_json_payload(self, metric_run: MetricRun):
+        result = {
+            "id": str(metric_run.id),
+            "metrics": [],
+        }
+        for metric in metric_run.metrics:
+            result["metrics"].append(
+                {
+                    "id": str(metric.id),
+                    "batch_id": metric.batch.id,
+                    "data_asset_id": str(metric.batch.data_asset.id),
+                    "metric_name": metric.metric_name,
+                    "exception_message": metric.exception.exception_message,
+                    "exception_type": metric.exception.exception_type,
+                    "value": metric.value,
+                    "metric_type": type(metric).__name__,
+                    "value_type": type(metric.value).__name__,
+                    "value_parameter_names": [],
```

Comment:
If we keep this field we will need to populate it here

Reply:
After thinking about this more, I think we will be removing this field.

Reply:
Removing `value_parameter_names`?

Reply:
Exactly. There are a few other changes in the confluence doc I sent earlier.